lib/TransportRPC.js

```diff
@@ -15,6 +15,13 @@ class TransportRPC extends TransportSocket {
   format (data) {
     return JSON.stringify(data)
   }
+
+  getSocket (port, conf) {
+    // https://github.com/uWebSockets/bindings/issues/23
+    throw new Error(
+      'implementation error: getSocket method needs to get extended'
+    )
+  }
```

Comment:
hmm, not super happy with `getSocket` but it also doesn't make sense to me to move it into the Peer*.js classes, as they are basically factories for the different transports...

🤔